contracts/extensions/contracts/src/MixinWethUtils.sol

```diff
@@ -80,37 +112,24 @@ contract MixinWeth is
         }
 
         // Convert remaining ETH to WETH.
-        ETHER_TOKEN.deposit.value(ethRemaining)();
+        WETH.deposit.value(ethRemaining)();
 
         return ethRemaining;
     }
 
-    /// @dev Refunds any excess ETH to msg.sender.
-    /// @param initialWethAmount Amount of WETH available after transferring affiliate fees.
-    /// @param wethSpent Amount of WETH spent when filling orders.
+    /// @dev Unwraps and refunds WETH to msg.sender.
+    /// @param refundAmount Amount of WETH balance to refund.
     function _transferEthRefund(
```

Comment:
Nit: now that this is in a more generic package, it probably makes sense to give this a more generic name. The first name that comes to mind is `withdrawAndTransferEth`.